src/main/java/com/hillel/items_exchange/service/ImageService.java

```diff
@@ -0,0 +1,33 @@
+package com.hillel.items_exchange.service;
+
+import com.hillel.items_exchange.dao.ImageRepository;
+import com.hillel.items_exchange.dto.ImageDto;
+import com.hillel.items_exchange.model.Image;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class ImageService {
+    private final ModelMapper modelMapper;
+    private final ImageRepository imageRepository;
+
+    public List<String> getLinksByProductId(Long id) {
+        return imageRepository.findByProductId(id).stream()
```

Comment:
You can create new method into repository that will extract only links from image it would be equal to SQL request:
``select links from image;``